clients/src/main/java/org/apache/kafka/clients/admin/internals/AdminMetadataManager.java

```diff
@@ -226,20 +225,30 @@ public void transitionToUpdatePending(long now) {
         this.lastMetadataFetchAttemptMs = now;
     }
 
+    public void updateFailed(Throwable exception) {
+        // We depend on pending calls to request another metadata update
+        this.state = State.QUIESCENT;
+
+        if (exception instanceof AuthenticationException) {
+            log.warn("Metadata update failed due to authentication error", exception);
+            this.authException = (AuthenticationException) exception;
+        } else {
+            log.debug("Metadata update failed", exception);
```

Comment:
Previously in `makeMetadataCall.handleFailure()` we will have INFO level log. Should we still keep the INFO level log here?